precheck/lib/precheck/runner.rb

```diff
@@ -185,7 +185,7 @@ def app
 
     def latest_app_version
       platform = Spaceship::ConnectAPI::Platform.map(Precheck.config[:platform])
-      @latest_version ||= app.get_edit_app_store_version(platform: platform)
+      @latest_version ||= app.get_latest_app_store_version(platform: platform)
```

Comment:
One more thing! Should this also use the `use_live`? Seems like we'd want the App Store version to clearly use either the "edit" or "live" version, right?

Reply:
Yup, nice catch! Updating now

Reply:
Done 💪